dist/Dockerfile

```diff
@@ -34,5 +42,12 @@ RUN apt-get -y install g++-multilib
 RUN apt-get -y install gcc-avr binutils-avr avr-libc
 RUN apt-get -y install subversion curl wget python p7zip unzip
 
+RUN wget http://launchpadlibrarian.net/206632429/ccache_3.2.2-2_amd64.deb -O /tmp/ccache_3.2.2-2_amd64.deb
+RUN dpkg -i /tmp/ccache_3.2.2-2_amd64.deb
+
+RUN mkdir -p /data/ccache
+RUN chmod 777 /data/ccache
+ENV CCACHE_DIR /data/ccache
```

Comment:
It might be beneficial to run multiple commands in the same RUN line in a Dockerfile. It reduces the number of image layers which may speed up launching and running the containers.

See also https://docs.docker.com/articles/dockerfile_best-practices/